requirements/constraints.txt

```diff
@@ -8,9 +8,8 @@
 # pin when possible.  Writing an issue against the offending project and
 # linking to it here is good.
 
-# Imposed by pytest-cookies 0.4.0 to continue supporting Python 3.4
+# Imposed by pytest-cookies 0.5.1 to continue supporting Python 3.4
 arrow<0.14.0
```

Comment:
Why do we still need the arrow constraint?

Reply:
The constraint is set by pytest-cookies, so we are stuck there till they drop py34 support,

Reply:
Ah, I see -- this is needed because it is depended on by both base.in and test.in dependencies, and the version resolution for base.in by itself is incompatible with a test dependency.